packages/gcc7.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Gcc7 < Package
+  description 'The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Ada, and Go.'
+  homepage 'https://www.gnu.org/software/gcc/'
+  version '7.3.0'
+
+  source_url 'https://github.com/gcc-mirror/gcc/archive/gcc-7_3_0-release.zip'
+
+  source_sha256 'd8cb199f0d98bd83e3dac645126722e44137c24cbe5e87e1113dd4270f7530c6'
+
+  depends_on 'buildessential'
+  depends_on 'binutils'
+  depends_on 'gmp'
+  depends_on 'mpfr'
+  depends_on 'mpc'
+  depends_on 'isl'
+  depends_on 'cloog'
+  depends_on 'glibc'
+  depends_on 'gawk'
+
+  def self.build
+    system "mkdir", "build"
+    Dir.chdir "build" do
+      ENV["LIBRARY_PATH"] = "/usr/local/lib64"
+      system "../configure", "-v", "--build=x86_64-linux-gnu", "--host=x86_64-linux-gnu", "--target=x86_64-linux-gnu", "--libdir=#{CREW_LIB_PREFIX}", "--prefix=#{CREW_PREFIX}", "--enable-checking=release", "--enable-languages=c,c++,fortran", "--disable-multilib", "--disable-libmpx", "--program-suffix=-7.3"
```

Comment:
Please put each option on a separate line to avoid horizontal scrolling.
```
../configure", "-v",
"--build=x86_64-linux-gnu",
"--host=x86_64-linux-gnu",
"--target=x86_64-linux-gnu",
"--libdir=#{CREW_LIB_PREFIX}",
"--prefix=#{CREW_PREFIX}",
"--enable-checking=release",
"--enable-languages=c,c++,fortran",
"--disable-multilib",
"--disable-libmpx",
"--program-suffix=-7.3"
```

Reply:
This also makes it easier to add/remove lines in the future.

Reply:
You could add:
```
"--build=#{ARCH}-linux-gnu",
"--host=#{ARCH}-linux-gnu",
"--target=#{ARCH}-linux-gnu",
```

Reply:
But excluding may be better, actually.

Reply:
Sounds good.  Ill remove them.